src/frontend/styles/components/writing.pcss

```diff
@@ -93,6 +93,10 @@
   .tc-cell {
     padding: 6px 8px;
   }
+
+  .ce-block__content {
```

Comment:
Looks strange, because it's important style for the editor.js layut

<img width="818" alt="image" src="https://user-images.githubusercontent.com/3684889/188727814-80109bd8-5b18-4ff9-99ca-a254354cf0e0.png">

Reply:
> The Content page and the Writing page should have the same width.

this requirement can not be achieved then. Content is 700px wide and editor blocks are 650px

Reply:
we can add extra paddings to content page, so that text itself is no wider that 650px

Reply:
You can override it instead of disabling:

```css
.ce-block__content,
.ce-toolbar__content {
  max-width: 700px;
}
```